app/models/health_status.php

```diff
@@ -244,12 +245,16 @@ function ($entity) use ($path) {
 }
 
 // Get stats
-$stats = Health::getStats($projects);
-$translated = $stats['translated'];
-$reference = $stats['total'];
-
-$completion = round(($translated / $reference) * 100, 2);
-$completion = $completion > 100 ? 100 : $completion;
+if (! empty($projects)) {
+    $stats = Health::getStats($projects);
+    $translated = $stats['translated'];
+    $reference = $stats['total'];
+    $completion = round(($translated / $reference) * 100, 2);
+    $completion = $completion > 100 ? 100 : $completion;
+} else {
+    $completion = 0;
+    error_log("ERROR: empty projects for {$locale}");
```

Comment:
Will need to remove this as soon as I understand how we get to an empty `$projects`